### PR TITLE
Ergo 114 transition select roles

### PR DESCRIPTION
**Pull request by bleto**

### CODE REVIEW

File: components/Multimedia/UserAvatar.vue [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+            type: String,
+            default: '',
+        },
+        pictureSize: {
+            type: Number,
+            default: 24,
+        },
+    },
+    computed: {
+        avatarInitial() {
```
Comment (derpdead) [inline components/Multimedia/UserAvatar.vue]: Not sure why is taht happening, shoudlw e get initials from be if there is no img?

File: components/Multimedia/UserAvatar.vue [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    props: {
+        imageId: {
+            type: String,
+            default: '',
+        },
+        userName: {
+            type: String,
+            default: '',
+        },
+        pictureSize: {
```
Comment (derpdead) [inline components/Multimedia/UserAvatar.vue]: picture size does not match component. You are using Avatar everywhere

File: components/Multimedia/UserAvatar.vue [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+            type: Number,
+            default: 24,
+        },
+    },
+    computed: {
+        avatarInitial() {
+            return this.userName.length ? this.userName[0] : '';
+        },
+        avatarStyles() {
```
Comment (derpdead) [inline components/Multimedia/UserAvatar.vue]: What about breaking convention with Classes, Styles at the end of the name if the return value is contextual - like here, you are setting an image bounds/size. Maybe lets simplify it to avatarSize?

File: components/NavigationBar/NavigationBarUserButton.vue [hunk 4]
```diff
@@ -7,7 +7,9 @@
         data-cy="user-select"
         @focus="onFocus">
         <template #input>
-            <UserPicture :image-id="user.avatar_id" />
+            <UserAvatar
+                :image-id="user.avatar_id"
+                :user-name="user.first_name" />
```
Comment (derpdead) [inline components/NavigationBar/NavigationBarUserButton.vue]: maybe just a name, title etc? No need to call it as a user-name

File: core/components/Multimedia/Picture.vue [hunk 5]
```diff
@@ -61,6 +66,12 @@ export default {
         max-height: 100%;
         object-fit: cover;
 
+        &--broke {
```
Comment (derpdead) [inline core/components/Multimedia/Picture.vue]: broken, typo

File: components/Multimedia/UserAvatar.vue (hunk 1 above)
Comment (bleto) [reply components/Multimedia/UserAvatar.vue]: yes, we get first letter from user name if we don't have avatar image.

**Pull request merged**